bool is_enem_left_marker_gotten
bool is_enem_right_marker_gotten
bool is_enem_back_marker_gotten
int32 enem_get_wall_marker_no